test/com/google/javascript/jscomp/ProcessCommonJSModulesTest.java

```diff
@@ -199,6 +199,43 @@ public void testModuleExportsScope() {
         "var module$test=foo$$module$test");
   }
 
+  public void testUMDPatternConversion() {
+    setFilename("test");
+    testModules(
+        "var foobar = {foo: 'bar'};" +
+        "if (typeof module === 'object' && module.exports) {" +
+        "module.exports = foobar;" +
```

Comment:
could you indent within the strings here? It's hard to read what this does. E.g. replace this line with: `      "    module.exports = ..." + ` (note the WS after the first quote).